src/components/leaflet/Leaflet_map.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { MapContainer, TileLayer, Marker,Popup } from 'react-leaflet';
+import style from './leaflet.module.scss';
+import Search from '../search_form/Search';
+
+const Leaflet_map = () => {
+  return (
+    <MapContainer className={style.mapCont} center={[51.505, -0.09]} zoom={13} scrollWheelZoom={false}>
```

Comment:
I don't think the map and the marker will always be the same. So let's move coordinates and a marker position to props.

Reply:
did it from App props